### PR TITLE
ui: fix dashboard on saml user login

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes: #8950
Fixes: #9045

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by shwstppr** [comment]
This works but will have to check how project was set in local storage earlier

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

**Comment by DaanHoogland** [comment]
@blueorangutan ui

**Comment by shwstppr** [comment]
@blueorangutan ui

File: ui/src/permission.js [hunk 1]
```diff
@@ -113,6 +113,10 @@ router.beforeEach((to, from, next) => {
                 next({ path: redirect })
               }
               const project = vueProps.$localStorage.get(CURRENT_PROJECT)
```
Comment (shwstppr) [inline ui/src/permission.js:115]:
```suggestion
              var project = vueProps.$localStorage.get(CURRENT_PROJECT)
```

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by weizhouapache — COMMENTED** [review]
code lgtm

needs testing

**Comment by rohityadavcloud** [comment]
LGTM - let me merge this and help test against the QA server.

**Pull request merged**